### PR TITLE
feat: Add embedding model and base url config for OpenAI provider

### PR DESCRIPTION
**Pull request by leehuwuj**

### CODE REVIEW

File: admin/sections/config/providers/openai.tsx [hunk 1]
```diff
@@ -44,10 +46,45 @@ export const OpenAIForm = ({
...
+      <div className="flex space-x-4">
+        <div className="w-1/2">
+          <ModelForm
+            form={form}
+            defaultValues={defaultValues}
+            supportedModels={supportingModels}
+          />
+        </div>
+        <div className="w-1/2">
+          <ModelForm
```
Comment (marcusschiesser) [inline admin/sections/config/providers/openai.tsx:58]: what's the default for embedding: `text-embedding-3-small`?

File: admin/sections/config/providers/openai.tsx [hunk 2]
```diff
@@ -44,10 +46,45 @@ export const OpenAIForm = ({
...
       />
-      <ModelForm
-        form={form}
-        defaultValues={defaultValues}
-        supportedModels={supportingModels}
+      <div className="flex space-x-4">
+        <div className="w-1/2">
+          <ModelForm
+            form={form}
+            defaultValues={defaultValues}
```
Comment (marcusschiesser) [inline admin/sections/config/providers/openai.tsx:53]: isn't it easier to just pass one value?
```suggestion
            defaultValue={defaultValues.model}
```

File: admin/sections/config/providers/shared.tsx [hunk 3]
```diff
@@ -14,22 +14,33 @@ import {
...
+  title,
+  description,
+  model_field_name,
   defaultValues,
   supportedModels,
 }: {
   form: UseFormReturn;
+  title?: string;
+  description?: string;
+  model_field_name?: string;
```
Comment (marcusschiesser) [inline admin/sections/config/providers/shared.tsx:33]: FormField's call this just `name` - how about?
```suggestion
  name?: string;
```

File: admin/sections/config/providers/openai.tsx (hunk 1 above)
Comment (leehuwuj) [reply admin/sections/config/providers/openai.tsx:58]: Yes it's. The default value is already specified in the client/providers

File: admin/sections/config/providers/shared.tsx [hunk 4]
```diff
@@ -14,25 +14,36 @@ import {
...
+  name,
+  defaultValue,
   supportedModels,
 }: {
   form: UseFormReturn;
-  defaultValues: any;
+  title?: string;
+  description?: string;
+  name?: string;
+  defaultValue: any;
```
Comment (marcusschiesser) [inline admin/sections/config/providers/shared.tsx:34]: nit: i guess that's a string

**Review by marcusschiesser — APPROVED** [review]

**Pull request merged**